main.js

```diff
@@ -3,6 +3,7 @@ const TrackedElement = require('./src/tracked-element');
 
 const tracked = [];
 let tracking = false;
+let bodyHeight = window.document.body.clientHeight;
```

Comment:
to be consistent, if you're referencing it via the document instance directly elsewhere, you probably don't need to use `window` here